CampusTour/ViewControllers/MainTabBarController.swift

```diff
@@ -0,0 +1,11 @@
+import UIKit
+
+class MainTabBarController: UITabBarController {
+    // empty parameter required to avoid infinite recursion. we should find a better solution for this.
+    convenience init(_: ()) {
```

Comment:
if the _ parameter is omitted, then this convenience init infinitely recurses. Not sure if there's a better workaround

Reply:
Hmm.. can't we do this in viewdidload?